docs/core/admin/limits.md

```diff
@@ -3,16 +3,43 @@ layout: doc
 title: Limits
 ---
 
-# Dovecot Built-in Limits
+# Dovecot Limits
 
-Dovecot contains various built-in resource limits designed to prevent
-denial of service situations. This page lists those limits.
+Dovecot contains various configurable and built-in resource limits designed to
+prevent denial of service situations. This page lists those limits.
 
 ::: todo
 This list is currently incomplete.
 :::
 
-## Message Headers
+## Configurable Limits
+
+Storage size limits:
+
+ * [[link,quota_root]
+ * [[setting,quota_max_mail_size]
+ * [[setting,quota_mailbox_count]
+ * [[setting,quota_mailbox_message_count]
+
+User concurrency limits:
+
+ * [[setting,lmtp_user_concurrency_limit]
+ * [[setting,mail_max_userip_connections]
```

Comment:
Just realized that all these links are broken.  `[[xyz]` instead of the correct `[[xyz]]`.

Reply:
At least these are now fixed.